src/screens/library_screen/library_screen.tsx

```diff
@@ -117,7 +136,12 @@ export class LibraryScreen extends Component<ownProps,ownState> {
 
     renderGridView() {
         const { arrayUserBooks, fetchUserBooksStatus } = this.props
+        const { searchQuery } = this.state
         let books = _.isEmpty(arrayUserBooks) ? [] : arrayUserBooks
+        books = _.isEmpty(searchQuery) ? 
+            books : 
+            _.filter(books, (book) => book.title.search(searchQuery) !== -1)
```

Comment:
What does this `-1` means? Please use a constant isntead that self explains the meaning of this value